dc-cudami-editor/src/components/WebpageList.jsx

```diff
@@ -10,103 +11,165 @@ import PublicationStatus from './PublicationStatus'
 import {formatDate} from './utils'
 import {typeToEndpointMapping} from '../api'
 
+const WebpageItem = forwardRef(
+  (
+    {
+      changeOfOrderActive,
+      enableMove,
+      enableRemove,
+      identifiable,
+      index,
+      language,
+      onMove,
+      onRemove,
+      pageNumber,
+      pageSize,
+      parentType,
+      showEdit,
+      style = {},
+      type,
+      widths,
+    },
+    ref
+  ) => {
+    const {
+      label,
+      lastModified,
+      previewImage,
+      previewImageRenderingHints,
+      publicationEnd,
+      publicationStart,
+      uuid,
+    } = identifiable
+    const {apiContextPath, uiLocale} = useContext(AppContext)
+    const viewBaseUrl = `${apiContextPath}${typeToEndpointMapping[type]}`
+    const publicationEndDate = publicationEnd && new Date(publicationEnd)
+    const publicationStartDate = publicationStart && new Date(publicationStart)
+    return (
+      <tr key={uuid} ref={ref} style={style}>
+        <td className="text-right" style={{width: widths[0]}}>
```

Comment:
nit: die ganzen `widths[index]` könnte man doch auch schön in Variablen umwandeln mit aussagekräftigen Beschreibungen: wie `index_col, publication_col, preview_col ... = widths` oder so.

Reply:
Danke für den Hinweis, schaut jetzt deutlich lesbarer aus :+1: